android/app/src/main/java/com/google/codeu/chatme/presenter/LoginActivityPresenter.java

```diff
@@ -92,7 +92,7 @@ public void onComplete(@NonNull Task<AuthResult> task) {
 
                             // saves new user to real-time database
                             addUser(currentUser.getUid(), currentUser.getDisplayName());
-                            view.openChatActivity();
+
```

Comment:
I think it would a good idea to open `TabsActivity` directly once an account is created. Are you thinking of something else?